src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package duke.tasks;
+
+import duke.exceptions.DukeExceptionIllegalArgument;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    ArrayList<Task> tasks; // composition
```

Comment:
Should tasks be private? It's not used outside of TaskList either

Reply:
(Y)